src/components/help-menu.tsx

```diff
@@ -0,0 +1,64 @@
+import { Badge } from "./ui/badge";
+import { Card } from "./ui/card";
+
+export default function HelpMenu() {
+  return (
+    <Card className="flex w-[350px] gap-2 p-2">
+      <div className="flex flex-col gap-2">
+        <Badge
```

Comment:
is it possible to replace these with the `StatusBadge` component so that everything is consistent?

Reply:
Yes. Updated:
<img width="429" alt="image" src="https://github.com/hack4impact-calpoly/ecologistics-portal/assets/81380688/6f845b44-2787-4698-9100-e81eb8fc4a1e">